pkg/asset/manifests/cloudproviderconfig.go

```diff
@@ -108,7 +108,11 @@ func (cpc *CloudProviderConfig) Generate(dependencies asset.Parents) error {
 		}
 		cm.Data[cloudProviderConfigDataKey] = azureConfig
 	case gcptypes.Name:
-		gcpConfig, err := gcpmanifests.CloudProviderConfig(clusterID.InfraID, installConfig.Config.GCP.ProjectID)
```

Comment:
why the new variable assignments?

also i'm more inclined towards explicit arguments rather than the function choosing the defaults here. but as-is behavior is also okay.

Reply:
>why the new variable assignments?

the line looks long, but I reverted this change. (resolved)

>also i'm more inclined towards explicit arguments rather than the function choosing the defaults here. but as-is behavior is also okay.

I think I have addressed  this point, but I wasn't exactly sure of what you meant so let me know if you meant/intended something else.